packages/manager/modules/bm-server-components/src/general-information/installation/ovh/server-installation-ovh.controller.js

```diff
@@ -1725,6 +1720,20 @@ export default class ServerInstallationOvhCtrl {
               )}`.replaceAll(/&#34;/g, '"');
   }
 
+  getDocumentationMessage(distribution) {
```

Comment:
Is this function used ? I don't find any usage

Reply:
yes it is used here: https://github.com/ovh/manager/pull/11721/files#diff-2883a93ed481a5ce68a86b235827cdbc281e8e24a053991354a763f79cdc3ba5R124